.github/dependabot.yaml

```diff
@@ -21,4 +21,4 @@ updates:
   - package-ecosystem: "docker"
     directory: "/"
     schedule:
-      interval: "weekly"
+      interval: "daily"
```

Comment:
Bumped this to "daily" just so we catch any UBI updates as soon as we can, to minimize the risk of rejection at release time.